src/webapp/pages/form-page/disease-outbreak-event/utils/updateDiseaseOutbreakEventFormState.ts

```diff
@@ -19,15 +20,25 @@ export function updateDiseaseOutbreakEventFormState(
 ): FormState {
     const updatedForm = updateFormStateAndApplySideEffects(prevFormState, updatedField);
 
+    const updatedFormWithRulesApplied =
+        diseaseOutbreakEventWithOptions.rules.filter(rule => rule.fieldId === updatedField.id)
+            .length > 0
+            ? applyRulesInFormState(
+                  updatedForm,
+                  updatedField,
+                  diseaseOutbreakEventWithOptions.rules
+              )
+            : updatedForm;
+
```

Comment:
[Subjective] if this kind of long expressions are broken down into pieces (local variable, extract rules, ...), the resulting expression formats more nicely and is more readable.